Makefile

```diff
@@ -335,11 +335,15 @@ ${ISTIO_OUT}/istioctl-osx: depend
 ${ISTIO_OUT}/istioctl-win.exe: depend
 	STATIC=0 GOOS=windows bin/gobuild.sh $@ ./istioctl/cmd/istioctl
 
-# generate the istioctl.bash completion file
+# generate the istioctl completion files
 ${ISTIO_OUT}/istioctl.bash: istioctl
 	${ISTIO_OUT}/istioctl collateral --bash && \
 	mv istioctl.bash ${ISTIO_OUT}/istioctl.bash
 
+${ISTIO_OUT}/_istioctl: istioctl
+	${ISTIO_OUT}/istioctl collateral --zsh && \
+	mv _istioctl ${ISTIO_OUT}/_istioctl
```

Comment:
zsh doesn't require it to be called `_istioctl` does it? If not we could rename it to `istioctl.zsh` so that it is more clear what it is an consistent with the bash one maybe

Reply:
Zsh completion functions for commands are stored in files with names beginning with an underscore _, and these files should be placed in a directory listed in the $fpath variable. This will be part of the next PR (part 3 is doc).